src/components/modals/ProgressModal.vue

```diff
@@ -36,14 +36,14 @@
         <div class="progress-form__right">
           <validation-provider v-slot="{ errors }" name="datetime" rules="required">
             <label class="form-group">
-              <span class="form-label">{{ $t('widget.history.time') }}</span>
+              <span class="form-label">{{ $t('widget.history.date') }}</span>
```

Comment:
Tenkte egentlig at vi kun endret dette i `KPIProgressModal`, men ikke for nøkkelresultat-modalen, der man faktisk setter tidspunkt. Men er kanskje greit med bare "Dato" uansett?

Reply:
Skal sies det også er headingen i tabellen uansett 😄 
<img width="394" alt="image" src="https://user-images.githubusercontent.com/2866225/205916320-ab5ebe64-7e9a-4177-bb14-aeb397025cdc.png">

Reply:
Ah, der er det faktisk tidspunkt fortsatt ja. Jeg hadde for meg at man valgte dato der og. :sweat_smile: Reverterer den delen.